mtags/src/main/scala-2/scala/meta/internal/pc/CompletionItemResolver.scala

```diff
@@ -40,7 +40,11 @@ class CompletionItemResolver(
               .filterNot(_.isEmpty)
               .toSeq
             item.setLabel(replaceScalaDefaultParams(item.getLabel, defaults))
-            if (metalsConfig.isCompletionItemDetailEnabled) {
+            if (
+              metalsConfig.isCompletionItemDetailEnabled && !item
+                .getDetail()
+                .isEmpty()
```

Comment:
I used `!` here just because I didn't want to convert java string -> scala string just for this check to get `nonEmpty`.